app/views/services/show.html.erb

```diff
@@ -1,134 +1,30 @@
+  <h2><%= fetch_entry_id(get_resource, @published_resource_name) %></h2>
+  <p class="subtitle"><%= display_header_subtitle(get_resource, @published_resource_name) %></p>
 <% end %>
 
-<% if @user_permissions == 'none' %>
-  <div class="eui-banner--danger">
-    <p class="eui-banner__message">
-      <%= "You don't have the appropriate permissions to #{service_action_text(@record_action)} this service." %>
-    </p>
-  </div>
-  <div class="no-access">
-    <h3 class="access-title">Access Denied</h3>
-    <p class="access-message">
-      It appears you do not have access to <%= service_action_text(@record_action) %> this content.<br />
-      If you feel you should have access, please check with your provider manager or ensure you are logged into the correct provider.
-    </p>
-  </div>
-<% elsif @user_permissions == 'wrong_provider' %>
-  <div class="eui-banner--warn">
-    <p class="eui-banner__message">
-      <%= link_to("You need to change your current provider to #{service_action_text(@record_action)} this service. Click here to change your provider.",
-        "#", id: "change-current-provider-banner-link",
-        data: { "provider": @provider_id, action_link: "change-provider-service-#{@record_action}" }) %>
-    </p>
-    <%= render_change_provider_service_action_link(@record_action, @concept_id, @revision_id) %>
-  </div>
-<% end %>
-
-<% if @unsupported_version %>
-  <%# with MMT-2301 see if this can be refactored into a partial %>
-  <div class="eui-banner--warn">
-    <p class="eui-banner__message">
-      <%= "This #{@published_resource_name} has been published to the CMR at a higher UMM version than what MMT currently supports. In order to prevent unintentional data loss, editing this #{@published_resource_name} is currently unavailable." %>
-    </p>
-    <p>
-      <%= "Cloning this #{@published_resource_name} to create a new draft record may result is loss of data." %>
-    </p>
-  </div>
-<% end %>
-
-<% if @unconfirmed_version %>
-  <%# with MMT-2301 see if this can be refactored into a partial. Also we should determine if we want not show any of the metadata record, or to show the preview but prevent certain links from working %>
-  <div class="eui-banner--warn">
-    <p class="eui-banner__message">
-      <%= "We cannot confirm the UMM version of this #{@published_resource_name}." %>
-    </p>
-  </div>
-  <div class="no-access">
-    <h3 class="access-title">Unconfirmed UMM Version</h3>
-    <p class="access-message">
-      <%= "It appears we currently cannot confirm the UMM version of this #{@published_resource_name}. In order to prevent unintentional data loss, editing this #{@published_resource_name} is currently unavailable." %>
-      Please try again or contact <%= mail_to('support@earthdata.nasa.gov', 'Earthdata Support') %>
-    </p>
-  </div>
-<% end %>
+<%= render partial: 'base_published_record/direct_access_without_permissions' %>
```

Comment:
I really like how many lines we're replacing with the partials

Reply:
I concur